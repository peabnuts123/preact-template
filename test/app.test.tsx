import { h } from "preact";
import { render } from "@testing-library/preact";

import App from "@app/app";

describe('App', () => {
  test('Matches snapshot', () => {
    // Setup
    const { baseElement } = render(<App />);

    // Test / Assert
    expect(baseElement).toMatchSnapshot();
  });
});

