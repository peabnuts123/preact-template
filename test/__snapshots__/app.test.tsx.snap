// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Matches snapshot 1`] = `
<body>
  <div>
    <div
      id="app"
    >
      <header
        class="header"
      >
        <h1>
          Preact App
        </h1>
        <nav>
          <a
            class="is-active"
            href="/"
          >
            Home
          </a>
          <a
            class=""
            href="/profile"
          >
            Me
          </a>
          <a
            class=""
            href="/profile/john"
          >
            John
          </a>
        </nav>
      </header>
      <div
        class="home"
      >
        <h1>
          Home
        </h1>
        <p>
          This is the Home component.
        </p>
      </div>
    </div>
  </div>
</body>
`;
